pegasus/sites.v3/code.org/public/maker/index.md.partial

```diff
@@ -93,7 +96,7 @@ Interested in incorporating physical computing, maker education, and/or robotics
   <h2>Subscribe for the latest news</h2>
   <p>Sign up for our monthly emails, which contain the latest news about tools, videos, and other important updates for Maker.</p>
   <div class="flex-container" style="text-align: center">
-    <iframe class="col-50" src="http://go.pardot.com/l/153401/2023-01-10/pqf2sr" width="100%" height="400" type="text/html" frameborder="0" allowtransparency="true" style="overflow:hidden; float: none"></iframe>
+    <iframe class="col-50" src="http://go.pardot.com/l/153401/2023-01-10/pqf2sr" scrolling="no" width="100%" height="400" type="text/html" frameborder="0" allowtransparency="true" style="overflow:hidden; float: none"></iframe>
```

Comment:
@dmcavoy I just checked the page on mobile and this was causing the form's Submit button to be hidden.

In Dropbox I changed the height to 450px and added back the scrolling — now it only scrolls on mobile and there's a reasonable amount of whitespace at the bottom of the page on desktop and tablet.

Reply:
Thank you for doing that Kelby. Good learning for me as well!